localization/english/wc_magic_l_english.yml

```diff
@@ -0,0 +1,9 @@
+﻿l_english:
+ soul_amount:0 "You currently have [GetPlayer.MakeScope.Var('collected_souls').GetValue|0V] Souls."
```

Comment:
Vanilla current value is written in a different style.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/140951761-edc7f51e-4e19-4f1e-8072-a095e026db66.png)

</details>